apertium-kaa.kaa.lexc

```diff
@@ -26565,13 +26571,13 @@ sóylemshek:sóylemshek V-IV-REFL ;
 sóyleniw:sóyleniw # ;
 sóylesiw:sóylesiw # ;
 sóyle:sóyle V-IV ;
-sóyle:sóyle V-TV ;
+sóyle:sóyle V-TV-CAUS ;
 sóyles:sóyles V-TV-COOP ;
 sóylestir:sóylestir V-TV-CAUS ;
 sóyletiw:sóyletiw # ;
 sóylet:sóylet V-TV-CAUS ;
 sóylewik:sóylewik A2 ;
-sóylew:sóylew # ;
+sóylew:sóylew V-IV-COOP ;
```

Comment:
The cooperative form is already in here (sóyles), but is entered wrong.  If sóylew should be categorised as anything other than a verbal noun / gerund form of a verb, then it should be a noun.